### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by MicrolemurDev** [4ian/GDevelop#2362] [Enhancement ideas (and their technical discussions)]
I noticed that the License for GDevelop is under MIT with no trademark usage. This is essentially what Apache 2.0 does and I would suggest licensing it under Apache 2.0 due to this. Thank you for your understanding.

**Comment by arthuro555** [comment]
1. If they are the same, why change? It is already explicitely stated that the Logo and name are property of 4ian, no need to get another lisence for that.
2. They are not the same, Apache is more restrictive. Why would anyone want GDevelop to be more restrictive?

**Comment by 4ian** [comment]
I would be interested to know more about any protection that Apache 2 can offer. 

> They are not the same, Apache is more restrictive.

Is it really? It's more verbose but is there anything protecting users or the software itself?
From what I read, Apache 2 license has:
- more verbose and explicit explanations about patents.
- rules about abuse of trademarks/product name ("This License does not grant permission to use the trade names, trademarks, service marks, or product names of the Licensor, except as required for reasonable and customary use in describing the origin of the Work and reproducing the content of the NOTICE file.") which is indeed equivalent to the terms added about usage of GDevelop logo and name.

**Comment by Aspie96** [comment]
> They are not the same, Apache is more restrictive. Why would anyone want GDevelop to be more restrictive?

That is true!

The Apache 2.0 license is not compatible with the OpenBSD criteria, for example. This might be OK, but shows it is a very different kind of license.

The MIT license doesn't provide any threadmark license anyways.

The Apache license, for instance, is not compatible with GPLv2.
